examples/mixture_density_network.py

```diff
@@ -82,5 +82,5 @@ def build_toy_dataset(nsample=6000):
                                 feed_dict={X: X_train, y: y_train})
     test_loss[i] = sess.run(inference.loss, feed_dict={X: X_test, y: y_test})
     print("Train Loss: {:0.3f}, Test Loss: {:0.3f}".format(train_loss[i], test_loss[i]))
-    pred_weights, pred_means, pred_std = sess.run(
+pred_weights, pred_means, pred_std = sess.run(
```

Comment:
Cool. That makes sense since we don't actually print anything about these predictions during training. 

Could you add an empty line between the end of the for loop and this line?